students/cindywaldron/lesson2/drawGrid.py

```diff
@@ -0,0 +1,21 @@
+def printGrid(x,y):
+	"""
+	Description: printGrid prints grids with given cell size
+	and grid size.                                   
+	:param: x - cell size
+	:param: y - grid size
+	"""
+	for i in range(x):
+		# construct string with pattern of '+', ' ', and '-'
```

Comment:
your indentation should be set to 4 spaces (no tabs)

please configure your editor to properly indent.